components/Scorecard.js

```diff
@@ -190,6 +190,8 @@ const Category = ({ category, expanded, onClickExpand, stateData }) => {
     stateData.resource_links.filter(
         (l) => l.category_id === category.id && l.active
     ) || [];
+    const categoryImageName = category.title.replace(/\(|\)/g, '').replace(/\s/g, '-').toLowerCase();
```

Comment:
This line takes the category title, strips any parentheses from it, replaces spaces with dashes, and lower cases it.